cmd/logcli/query.go

```diff
@@ -21,9 +22,8 @@ func doQuery() {
 	}
 
 	var (
-		i            iter.EntryIterator
-		common       labels.Labels
-		maxLabelsLen = 100
```

Comment:
Just remove it, how about put it into config?

Reply:
There is no config file for logcli, I'm not following you.